src/pricing.js

```diff
@@ -7,6 +7,7 @@ var Property = require('./base').Property;
 var Field = require('./base').Field;
 
 var pricing_common = require('./pricing_common');
+var Price = pricing_common.Price;
```

Comment:
According to http://developer.oanda.com/rest-live-v20/pricing-ep/, this should be `ClientPrice`, not `Price`.

Note that the price stream endpoint returns the `time` field which class `ClientPrice` supports, but class `Price` does not.